crates/optimism/txpool/src/maintain.rs

```diff
@@ -43,6 +43,11 @@ struct MaintainPoolInteropMetrics {
     /// Counter indicating the number of conditional transactions removed from
     /// the pool because of exceeded block attributes.
     removed_tx_interop: Counter,
+    /// Number of interop transactions currently in the pool
+    interop_txs_in_pool: Counter,
```

Comment:
this should be a `Gauge` because we need to track concrete values.

let's also rename this to `pooled_interop_transactions`